File: C/6kyu/Is a Number Prime/solution.c
#include <stdbool.h>
#include <math.h>

bool is_prime(int num)
{
    if(num <= 1)
      return false;
      
    int max = sqrt(num), ignored[max], freeSpace = 0;
    
    for(int i = 2; i <= max; i++) {
      if(num % i == 0)
        return false;
    }
    
    return true;
}

File: PHP/6kyu/Is a Number Prime/solution.php
<?php
function is_prime(int $n): bool {
  if($n > 1) {
    for($i = 2; $i <= sqrt($n); $i++){
      if($n % $i == 0)
        return false;
    }
    return true;
  }
  return false;
}
?>